CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ _See also: [the official LibGDX changelog](https://github.com/libgdx/libgdx/blob
 
 #### 1.10.0-SNAPSHOT
 
+- **[FEATURE]** Implemented factory methods for LibGDX arrays of primitives
```

Comment:
I usually place code-related changes below `[UPDATE]` entries and mention which module it affects. It's a minor change that I will correct shortly, since it's unrelated to the actual library code.

Reply:
@czyzby thanks for your reply, will keep in mind this advice in case of future PRs

Reply:
Thanks. You can see the minor changes that I applied [in this commit](https://github.com/libktx/ktx/commit/95353dfa629fb0705f426350b571613cf5539cca).